src/Raven.Server/Documents/DatabasesLandlord.cs

```diff
@@ -47,6 +47,20 @@ public DatabasesLandlord(ServerStore serverStore)
             _concurrentDatabaseLoadTimeout = _serverStore.Configuration.Databases.ConcurrentLoadTimeout.AsTimeSpan;
             _logger = LoggingSource.Instance.GetLogger<DatabasesLandlord>("Server");
             CatastrophicFailureHandler = new CatastrophicFailureHandler(this, _serverStore);
+            PoolOfThreads.GlobalRavenThreadPool.LongRunning(_ =>
```

Comment:
I want to have a single task per server and not per database, otherwise not sure about the performance of this when we will have let say 50 databases..

Reply:
Why? With a single server wide task, you are causing convoys. 
Better to have one per database and have them handle it directly without one db interfering with another